### ISSUE TITLE
Single-source crate dependency versions

### ISSUE THREAD

**Issue by neysofu** [graphprotocol/graph-node#4013]
Would be nice to use the workspace inheritance feature introduced with Rust 1.64 to provide a single source of truth for dependency versions.

Blocked by https://github.com/dependabot/dependabot-core/issues/5315.

**Comment by poliorcetics** [comment]
I opened a PR for it https://github.com/dependabot/dependabot-core/pull/5794, it should work but it's not merged yet.

It's probably possible to regenerate the docker container and use it locally to make the necessary changes but I have no idea how to do that. I'll gladly accept people testing it and telling me if it works has expected in a variety of configs though 👍

**Comment by neysofu** [comment]
Thanks @poliorcetics! I see your PR got merged, nice! Is the feature available already, then?

**Comment by poliorcetics** [comment]
It should be. I found a bug, but the use case to encounter is not something I have seen in practice, you can try it out and see if you have no dependabot mrs anymore

**Comment by neysofu** [comment]
Yeah, saw the new issue but that use case doesn't apply to us, so we're unblocked here. Thank you for the PR!

**Comment by neysofu** [comment]
Closed by #4061.
